pages/register.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import Head from 'next/head'
+import RegisterForm from '../components/RegisterForm'
+import styles from '../styles/Login.module.css'
+import router from 'next/router'
+
+const Register = () => { 
+  const registerRequest = (formData: React.FormEvent<HTMLInputElement>) => {

```

Comment:
Please move your API functions in the `api` folder. Something like `api/sessions`. 

This file will then hold all your fetch requests that relate to the session

```
export const register = () => {}

export const login = () => {}
```